scripts/deploy.ts

```diff
@@ -29,28 +28,30 @@ async function main() {
     3: 'ropsten',
     4: 'rinkeby',
     5: 'goerli',
+    80001: 'mumbai',
   } as { [chainId: number]: string }
   const chainName = chains[chainId]
 
-  const contractName = 'SealCredPaymaster'
+  const contractName = 'BWLPaymaster'
```

Comment:
Let's extract this, so you don't need to change it here and in `test/BWLPaymaster.ts` each time

Reply:
Something like `data/contractName.ts`

Reply:
This is not necessary, we have a name in the contract and we still have to change it